doc/can-define-object.md

```diff
@@ -96,6 +96,27 @@ myInstance.prop = "NEW VALUE";
 ```
 @codepen
 
+#### Observable class fields
+
+`ObservableObject` [class fields](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Class_fields) are observables like [can-observable-object/object.static.props static props]:
```

Comment:
I think this should be good:

> ObservableObject class fields are also observable.

Reply:
@phillipskevin The docs are updated, can you review one more time?